

<launch> 
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" > 
    <param name="video_device" value="/dev/video0" /> 
    <param name="image_width" value="640" /> 
    <param name="image_height" value="480" /> 

  </node>
  <!-- <node name="cv_camera" pkg="cv_camera" type="cv_camera_node" output="screen" > 
    <param name="video_device" value="/dev/video1" /> 
    <param name="image_width" value="100" /> 
    <param name="image_height" value="50" /> 
  </node>  -->

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen"> 
    
    <remap from="image" to="/usb_cam/image_raw"/> 
    <param name="autosize" value="true" /> 
  </node> 
</launch>
<!-- 
<launch>
  <arg name="image_view" default="true" />

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
     <rosparam command="load" file="$(find usb_cam)/config/usb_cam.yml"/> 
      <param name="image_width" value="640" /> 
      <param name="image_height" value="480" /> 
  </node>
  <node if="$(arg image_view)" name="image_view" pkg="image_view" type="image_view"
        respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node> -->

  <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen"> 
    <remap from="image" to="/jackal0/front/left/image_raw"/> 
    <param name="autosize" value="true" /> 
  </node>  
</launch>-->
